config.rb

```diff
@@ -40,3 +40,5 @@
 activate :livereload
 activate :neat
 activate :syntax
+
+page '/feed.xml', layout: false
```

Comment:
![](http://dl.dropboxusercontent.com/u/86731380/gifs/no_no_no_no_no.gif)


I followed the existing style. Happy to update if the maintainer prefers though.